src/broker/Manager.cc

```diff
@@ -297,6 +297,36 @@ void Manager::DoInitPostScript() {
     options.disable_forwarding = ! get_option("Broker::forward_messages")->AsBool();
     options.use_real_time = use_real_time;
 
+    options.peer_buffer_size = get_option("Broker::peer_buffer_size")->AsCount();
+    auto peer_overflow_policy = get_option("Broker::peer_overflow_policy")->AsString()->CheckString();
```

Comment:
Nit, could use `ToStdString()` and just  `==` below rather than `util::streq`.

Reply:
Could you live with doing that in a follow-up? Dominik likely did that just to stay consistent with the rest of the use in that function, and I'm wondering whether we should just ditch `get_option()` in favor of `id::find_const()`, or some such. What do you think? I can tuck it on here if you prefer.

Reply:
Not a biggie!